src/main/java/marquez/common/models/ConnectionUrl.java

```diff
@@ -22,12 +22,12 @@
 @EqualsAndHashCode
 @ToString
 public final class ConnectionUrl {
-  @Getter private final DataSource dataSource;
```

Comment:
We should renamed the class `DataSource` in seperate PRs. Going forward, we'll need to have an issue. Mind still crating one for the name change and link it in your PR?

Reply:
Sure - filed #360.

Reply:
We've settled on the class naming to be `Datasource` - should the variable naming also be `datasource`?

Reply:
Yes, for consistency

Reply:
Please see PRs #394, #387, and #384, where this is fixed.